Model/SyncStatus.php

```diff
@@ -98,7 +99,7 @@ public function sync($order)
      */
     private function markPaymentSuccessful($order, $charge)
     {
-        if ($charge['refunds'] && $order->getState() != Order::STATE_CLOSED) {
+        if ($charge['refunds']['data'] && $order->getState() != Order::STATE_CLOSED) {
```

Comment:
This line can throw error if `refunds` keyword is not in `$charge`

Reply:
Thanks for pointing this. Although `refunds` exists in each response, it is better to add a check.